calculator/calculator.js

```diff
@@ -23,9 +23,7 @@ const power = function(a, b) {
 const factorial = function(n) {
   if (n === 0) return 1;
   let product = 1;
-  for (let i = n; i > 0; i--) {
-    product *= i;
-  }
+  for (n; n > 0; n--) product *= n;
```

Comment:
Even though this works, and it might be more preference, but I think we should keep the curly braces in. If it has been mentioned leading up to this exercise about omitting curly braces in certain situations, it could be worth adding a comment to show the alternate method of without braces.

Reply:
@alejandroh9800 just wanted to check if you had seen the above comment.